js/apps/admin-ui/src/user/EditUser.tsx

```diff
@@ -58,80 +57,21 @@ export default function EditUser() {
   const [bruteForced, setBruteForced] = useState<BruteForced>();
   const [refreshCount, setRefreshCount] = useState(0);
   const refresh = () => setRefreshCount((count) => count + 1);
-
-  useFetch(
-    async () => {
-      const [user, currentRealm, attackDetection] = await Promise.all([
-        adminClient.users.findOne({ id: id!, userProfileMetadata: true }),
-        adminClient.realms.findOne({ realm }),
-        adminClient.attackDetection.findOne({ id: id! }),
-      ]);
-
-      if (!user || !currentRealm || !attackDetection) {
-        throw new Error(t("common:notFound"));
-      }
-
-      const isBruteForceProtected = currentRealm.bruteForceProtected;
-      const isLocked = isBruteForceProtected && attackDetection.disabled;
-
-      return { user, bruteForced: { isBruteForceProtected, isLocked } };
-    },
-    ({ user, bruteForced }) => {
-      setUser(user);
-      setBruteForced(bruteForced);
-    },
-    [refreshCount],
-  );
-
-  if (!user || !bruteForced) {
-    return <KeycloakSpinner />;
-  }
-
-  return (
-    <EditUserForm user={user} bruteForced={bruteForced} refresh={refresh} />
-  );
-}
-
-type EditUserFormProps = {
-  user: UserRepresentation;
-  bruteForced: BruteForced;
-  refresh: () => void;
-};
-
-const EditUserForm = ({ user, bruteForced, refresh }: EditUserFormProps) => {
```

Comment:
Why get rid of the separate component for the form?

Reply:
because it didn't update the user with this `useUpdateEffect`